apps/docs/uno.config.ts

```diff
@@ -0,0 +1,8 @@
+import { defineConfig, PresetOrFactory, UserConfig } from "unocss";
+import { presetHv } from "@hitachivantara/uikit-uno-preset";
+
+const config: UserConfig = {
+  presets: [presetHv() as PresetOrFactory<object>],
+};
+
+export default defineConfig(config);
```

Comment:
`UserConfig` is widening the types unnecessarily. If we just inline it, the type issue goes away

```suggestion
export default defineConfig({
  presets: [presetHv() as PresetOrFactory<object>],
});
```

Reply:
Changed to .mjs for consistency